references.bib

```diff
@@ -207,11 +213,13 @@ @techreport{kang1990feature
     year        = {1990},
     institution = {Carnegie-Mellon Univ Pittsburgh Pa Software Engineering Inst}
   }
-  @inproceedings{hu2021formalizing,
+  @misc{hu2021formalizing,
```

Comment:
You should not be using 'misc' to much - this is a hacky way to fix things. This is a proceedings paper, you should leave it like that.